mgmt/rest/server.go

```diff
@@ -80,6 +80,38 @@ type Config struct {
 	RestAuthPassword string `json:"rest_auth_password,omitempty"yaml:"rest_auth_password,omitempty"`
 }
 
+const (
+	CONFIG_CONSTRAINTS = `
+	"restapi" : {
+					"type": ["object", "null"],
+					"properties" : {
+									"enable": {
+													"type": "boolean"
+									},
+									"https" : {
+													"type": "boolean"
+									},
+									"rest_auth": {
+													"type": "boolean"
+									},
+									"rest_auth_password": {
+													"type": "string"
+									},
+									"rest_certificate": {
+													"type": "string"
+									},
+									"rest_key" : {
+													"type": "string"
+									},
+									"port" : {
+													"type": "integer"
```

Comment:
You are right. It would be a weird thing to do but since we output the port selected no reason to prevent it.